unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/AvatarModifierArea.cs

```diff
@@ -117,10 +129,13 @@ private HashSet<GameObject> DetectAllAvatarsInArea()
 
         Vector3 center = entity.gameObject.transform.position;
         Quaternion rotation = entity.gameObject.transform.rotation;
-        return cachedModel.area?.DetectAvatars(center, rotation);
+        return cachedModel.area?.DetectAvatars(center, rotation, excludedColliders);
```

Comment:
I would inverse the logic here. It feels more natural to have a function like `DetectAvatars(center, rotation, avatars)` than an exclusion list

Reply:
Uhmmm I think in the end that would be almost the same, because the provided `avatars` parameter  value would be like an "allowed avatars", and would also have to be checked on every collider found in the trigger area. I find it more natural that the `DetectAvatars` detects everything (not only the allowed avatars) and filters out the excluded ones, as it is in the current implementation, but it's just my opinion

Reply:
Having an "allow list" would make the code more complex:
1- for backward compatibility we will need to check if the allow list exists in the component and that it is valid.
2- since the sdk component is designed to have an exclusion list, we will need to convert that exclusion list into an allow list every time an avatar shape is instantiated.
we decided to have an exclusion list since it's the most common use case, to avoid, in most scenarios, the scene dev to keep track of every avatar that is inside the scene and modify the list accordingly constantly. This also means that the component would be updated less frequently

Reply:
I'm just talking about the `DetectAvatars` function. Everything would be the same but that function receive the Avatar colliders to detect in that area.

You just get all the avatars, remove the exclusion list and pass it to the area to detect

Reply:
`DetectAvatars` does a collision test with everything in `AVATAR_TRIGGER_LAYER` layer
we are not handling any list of avatar's colliders at this point
except for filtering that collision test result with `excludedColliders`

Reply:
@pbosio and @AjimenezDCL agreed on this feedback not applying for this PR. Marking this conversation as solved.